assets/app/styles/_components.less

```diff
@@ -202,40 +201,69 @@
       .osc-object-highlight();
       .flex(@columns: 1 1 auto);
       max-width: 450px;
+      min-width: 90px;
       border-radius: @border-radius-md;
-      border: 1px solid transparent;
-      // box-shadow: @osc-object-box-shadow; <- remove if it's not used in highlight
+      border: 1px solid #fff;
       padding: 10px;
       position: relative;
       margin: 0 10px 10px 0;
       text-align: center;
-      background-color: @pod-pending-bg;
+      background-color: #fcfcfc;
+      background-image: repeating-linear-gradient(-45deg, transparent, transparent 6px, #eeeeee 6px, #eeeeee 12px);
```

Comment:
is this for the new/pending/unknown states? I'd rather see this under a qualifier, instead of the default that other states have to undo